example-scripts/00-utils/httpTransport.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    JsonRpcRequest,
+    RequestPayload,
+    ResponsePayload,
+    RpcTransport,
+} from 'core-types'
+import { v4 as uuidv4 } from 'uuid'
+
+export class HttpTransport implements RpcTransport {
```

Comment:
this is duplicated from the implementation in the remote client. I think we should publish our RpcTransport implementations (Marc is also working on one for the browser Window postMessage) to a separate package for wallet kit consumers to use